src/components/Tool/MergeContacts/MergeContacts.test.tsx

```diff
@@ -136,10 +144,38 @@ describe('Tools - MergeContacts', () => {
     userEvent.click(queryAllByTestId('rightButton')[0]);
     expect(await findByText('Use this one')).toBeInTheDocument();
     userEvent.click(queryAllByTestId('ignoreButton')[0]);
+    userEvent.click(queryAllByTestId('ignoreButton')[1]);
     expect(queryByText('Use this one')).not.toBeInTheDocument();
+
+    userEvent.click(getByRole('button', { name: 'Confirm and Continue' }));
+    await waitFor(() =>
+      expect(mockEnqueue).toHaveBeenCalledWith('Updated 2 duplicate(s)', {
+        variant: 'success',
+      }),
+    );
+
+    const mergeCalls = mutationSpy.mock.calls
+      .map(([{ operation }]) => operation)
+      .filter(({ operationName }) => operationName === 'MassActionsMerge');
+    expect(mergeCalls).toHaveLength(0);
+
+    const ignoreCalls = mutationSpy.mock.calls
+      .map(([{ operation }]) => operation)
+      .filter(({ operationName }) => operationName === 'UpdateDuplicate');
+    expect(ignoreCalls).toHaveLength(2);
+    expect(ignoreCalls[0].variables).toEqual({
+      input: {
+        attributes: {
+          ignore: true,
+        },
+        type: TypeEnum.Contact,
+        id: '1',
+      },
+    });
+    expect(queryByTestId('ignoreButton')).not.toBeInTheDocument();
     expect(
-      getByRole('button', { name: 'Confirm and Continue' }),
-    ).not.toBeDisabled();
+      getByText('No duplicate contacts need attention'),
+    ).toBeInTheDocument();
```

Comment:
On my local I had some issues with the incorrect contact IDs being set to merge / delete. After setting record 1 to be deleted, when switching to record 2 to be deleted, it wouldn't update the actions array.

Please can you add soe tests her for that, as that it critial for this to work.